site/assets/scss/_syntax.scss

```diff
@@ -1,108 +1,123 @@
-/* KeywordPseudo */ .chroma .kp { color: $orange-700; }
-/* KeywordReserved */ .chroma .kr { color: $orange-700; }
-/* KeywordType */ .chroma .kt { color: $cyan-700; }
-/* Literal .chroma .l { } */
-/* LiteralDate .chroma .ld { color: $orange-700 } */
-/* LiteralNumber */ .chroma .m { color: $green-700; }
-/* LiteralNumberBin */ .chroma .mb { color: $teal-700; }
-/* LiteralNumberFloat */ .chroma .mf { color: $green-700; }
-/* LiteralNumberHex */ .chroma .mh { color: $green-700; }
-/* LiteralNumberInteger */ .chroma .mi { color: $green-700; }
-/* LiteralNumberIntegerLong */ .chroma .il { color: $green-700; }
-/* LiteralNumberOct */ .chroma .mo { color: $green-700; }
-/* LiteralString */ .chroma .s { color: $red-500; }
-/* LiteralStringAffix */ .chroma .sa { color: $blue-600; }
-/* LiteralStringBacktick */ .chroma .sb { color: $red-500; }
-/* LiteralStringChar */ .chroma .sc { color: $red-500; }
-/* LiteralStringDelimiter */ .chroma .dl { color: $blue-600; }
-/* LiteralStringDoc */ .chroma .sd { font-style: italic; color: $red-500; }
-/* LiteralStringDouble */ .chroma .s2 { color: $red-500; }
-/* LiteralStringEscape */ .chroma .se { color: $red-500; }
-/* LiteralStringHeredoc */ .chroma .sh { color: $red-500; }
-/* LiteralStringInterpol */ .chroma .si { color: $red-600; }
-/* LiteralStringOther */ .chroma .sx { color: $red-500; }
-/* LiteralStringRegex */ .chroma .sr { color: $teal-700; }
-/* LiteralStringSingle */ .chroma .s1 { color: $red-500; }
-/* LiteralStringSymbol */ .chroma .ss { color: $yellow-800; }
-/* Name */ .chroma .n { color: $green-700; }
-/* NameAttribute */ .chroma .na { color: $green-700; }
-/* NameBuiltin */ .chroma .nb { color: $green-700; }
-/* NameBuiltinPseudo .chroma .bp { } */
-/* NameClass */ .chroma .nc { color: $blue-600; }
-/* NameClass */ .chroma [class*="css"] .nc { font-weight: $font-weight-bold; }
-/* NameConstant */ .chroma .no { color: $green-700; }
-/* NameDecorator */ .chroma .nd { color: $purple-700; }
-/* NameEntity */ .chroma .ni { color: $gray-800; }
-/* NameException */ .chroma .ne { color: $red-500; }
-/* NameFunction */ .chroma .nf { color: $teal-700; }
-/* NameFunctionMagic .chroma .fm { } */
-/* NameLabel */ .chroma .nl { color: $purple-700; }
-/* NameNamespace */ .chroma .nn { color: $red-500; }
-/* NameOther */ .chroma .nx { color: $purple-700; }
-/* NameProperty .chroma .py { } */
-/* NameTag */ .chroma .nt { color: $blue-600; }
-/* NameTag */ .chroma .language-html .nt,
-.chroma .language-css .nt { font-weight: $font-weight-bold; }
-/* NameVariable */ .chroma .nv { color: $red-500; }
-/* NameVariableClass .chroma .vc { } */
-/* NameVariableGlobal .chroma .vg { } */
-/* NameVariableInstance .chroma .vi { } */
-/* NameVariableMagic .chroma .vm { } */
-/* Operator */ .chroma .o { color: $gray-700; }
-/* OperatorWord */ .chroma .ow { color: $black; }
-/* Punctuation */ .chroma .p { color: $gray-700; }
-/* TextWhitespace */ .chroma .w { color: $gray-400; }
+:root {
```

Comment:
I've randomly ordered the `--baseXX` colors based on my migration. I can try to regroup some of them but I don't know exactly in which order. Ideas?